backend/tests/test_wfh_mission_cli.py

```
@@ -0,0 +1,101 @@
+    proc = _run(control, "resume", "--intent", "ادامه کار گروهی", "--json", "--observed-main-sha", "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb")
+
+    assert proc.returncode == 0, proc.stderr
+    output = json.loads(proc.stdout)
+    assert output["disposition"] == "RESUME_READY"
+    assert output["checkpoint"]["mission_id"] == "WFH-ME-V3-20260902"
+    assert output["checkpoint"]["mission_state"]["next_action"] == "continue Task 5"
+
+
+def test_cli_normalizes_unicode_whitespace_for_canonical_phrase(tmp_path: Path) -> None:
+    control = _control_root(tmp_path)
+    proc = _run(control, "resume", "--intent", "  ادامه   کار\tگروهی  ", "--json", "--observed-main-sha", "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb")
```

Ruff (0.16.3)

[warning] String contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)